lib/internal/modules/esm/translators.js

```diff
@@ -323,6 +331,16 @@ function createCJSModuleWrap(url, source, isMain, loadCJS = loadCJSModule) {
 
 }
 
+translators.set('commonjs-sync', function importCommonJS(url, source, isMain) {
```

Comment:
Would this function be used for `import`ed CJS modules?
```suggestion
translators.set('commonjs-sync', function requireCommonJS(url, source, isMain) {
```